pkg/manager/manager.go

```diff
@@ -142,8 +142,10 @@ func (s *singleton) PostInit() error {
 	// clear the downloads and tmp directories
 	// #1021 - only clear these directories if the generated folder is non-empty
 	if s.Config.GetGeneratedPath() != "" {
+		logger.Info("Please wait. Deleting temporary files...")
```

Comment:
I don't think this should be logged on every startup. I would perform the `EmptyDir` actions in a goroutine and wait for it to finish. If it takes more than 5 seconds or so, log a message indicating that it is deleting the temp files.